src/main/java/frc/robot/subsystems/Dashboard.java

```diff
@@ -107,6 +109,23 @@ private Dashboard()
     
         _autonomousLog = autonomousOptions.add("Auto Log", "").withPosition(0, 4).withSize(1, 1).withWidget(BuiltInWidgets.kTextView).getEntry();
 
+        //Preferences
+        Preferences.getDouble("FL", 90.0);
+        Preferences.getDouble("FR", 90.0);
+        Preferences.getDouble("BL", 90.0);
+        Preferences.getDouble("BR", 90.0);
+
+        Preferences.getDouble("Max Extend", 36.0);
+        Preferences.getDouble("Arm Min", -90.0);
+        Preferences.getDouble("Arm Max", 90.0);
+
+        Preferences.getDouble("Wrist Min", -90.0);
+        Preferences.getDouble("Wrist Max", 90.0);
+        Preferences.getDouble("Twist Min", -180.0);
+        Preferences.getDouble("Twist Max", 180.0);
+        Preferences.getDouble("Eject Time", 1.0);
+        Preferences.getDouble("Intake Speed", 0.75);
```

Comment:
These lines don't actually do anything. From what I can see in the WPILib docs, `getDouble` returns a value from Preferences if the corresponding key exists in Preferences but doesn't add it if it doesn't exist. You're also not doing anything with the return value of the method, meaning these lines don't do anything at all and can be removed.